resources/etc/nginx/include.d/warp_analytics.conf.tpl

```diff
@@ -18,6 +18,12 @@ if ($http_x_requested_with ~ XMLHttpRequest) {
 	set $warp '</body>';
 }
 
+# Include google analytics scripts if a tracking id is set
+{{ if .Config.Exists "trackingID" }}
```

Comment:
The etcd key should be more specific to Google Analytics, perhaps we want to add support for other tracking platforms like Piwik.